src/_includes/series.html

```diff
@@ -17,6 +17,7 @@
 
     {%- comment -%} Add release date to game listed if present {%- endcomment -%}
     {%- assign game_data = include.series.games[firstGame] -%}
+    {%- assign release_year = game_data.release_date | split: "-" | first -%}
     {%- if game_data.release_year != null -%}
         {%- assign release_year_suffix = ' <span class="text-muted fs-5">(RELEASE)</span>' | replace: "RELEASE", game_data.release_year -%}
```

Comment:
```suggestion
        {%- assign release_year_suffix = ' <span class="text-muted fs-5">(RELEASE)</span>' | replace: "RELEASE", release_year -%}
```

Reply:
there's actually a few occurrences for release_year

![image](https://github.com/user-attachments/assets/da3aea81-a31d-4094-96ca-d983a9144316)